apps/wallet/src/ui/app/pages/dapp-tx-approval/index.tsx

```diff
@@ -208,10 +222,11 @@ export function DappTxApprovalPage() {
     }[] = useMemo(() => {
         switch (txRequest?.tx.type) {
             case 'v2': {
+                const data = txRequest.tx.data;
                 return [
                     {
                         label: 'Transaction Type',
-                        content: txRequest.tx.data.kind,
+                        content: data.kind,
```

Comment:
Remove this, we don't want this to say "v2" in the UI.

Reply:
done